data/i18n/ar/ar.toml

```
@@ -48,10 +48,10 @@ other = "و"
 other = "رمز تعريف الCVE"
 
 [cve_issue_url]
-other = "رابط القضية للCVE"
+other = "رابط البلاغ للCVE"
 
 [cve_summary]
-other = "ملخص القضية"
+other = "ملخص البلاغ
```

The missing ending quote `"` is causing a build failure ([check build logs here](https://app.netlify.com/sites/kubernetes-io-ar-staging/deploys/66f0c19684edb00008d0b374)).
I guess the intention was to retain the quotes and just change the text.
 
 ```suggestion
other = "ملخص البلاغ"
```